mkdocs.yml

```
@@ -0,0 +1,82 @@
+site_name: AWS SDK for Python
+site_description: Documentation for AWS SDK for Python Clients
+
+copyright: Copyright &copy; 2025, Amazon Web Services, Inc
+repo_name: awslabs/aws-sdk-python
+repo_url: https://github.com/awslabs/aws-sdk-python
+
+theme:
+  name: material
+  palette:
+    - scheme: default
+      primary: white
+      accent: light blue
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - scheme: slate
+      primary: black
+      accent: light blue
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+  features:
+    - navigation.tabs
+    - navigation.tabs.sticky
+    - navigation.top
+    - search.suggest
+    - search.highlight
+    - content.code.copy
+
+plugins:
+  - search
+  - gen-files:
+      scripts:
+        - scripts/generate_doc_stubs.py
+        - scripts/generate_nav.py
+  - mkdocstrings:
+      handlers:
+        python:
+          options:
+            show_source: false
+            show_signature: true
+            show_signature_annotations: true
+            show_root_heading: true
+            show_root_full_path: false
+            show_object_full_path: false
+            show_symbol_type_heading: true
+            show_symbol_type_toc: true
+            show_category_heading: true
+            group_by_category: true
+            separate_signature: true
+            signature_crossrefs: true
+            filters:
+            - "!^_"
+            - "!^deserialize"
+            - "!^serialize"
+
+markdown_extensions:
+  - pymdownx.highlight
+  - pymdownx.inlinehilite
+  - pymdownx.snippets:
+      base_path: ['.']
```

```suggestion
      base_path: ['.']
      check_paths: true
```

I think we should set `check_paths: true` which will _"Make the build fail if a snippet can't be found."_

Reference: https://facelessuser.github.io/pymdown-extensions/extensions/snippets/#options

---

Added `check_paths: true`. Removed `base_path: ['.']` since it is the default.